app/backend/approaches/chatreadretrieveread.py

```diff
@@ -57,7 +57,7 @@ def __init__(
     def system_message_chat_conversation(self):
         return """Assistant helps the company employees with their healthcare plan questions, and questions about the employee handbook. Be brief in your answers.
         Answer ONLY with the facts listed in the list of sources below. If there isn't enough information below, say you don't know. Do not generate answers that don't use the sources below. If asking a clarifying question to the user would help, ask the question.
-        For tabular information return it as an html table. Do not return markdown format. If the question is not in English, answer in the language used in the question.
+        If the question is not in English, answer in the language used in the question.
```

Comment:
Did we need to explicitly ask for markdown in the prompt?

Reply:
In my experience, LLMs tend to produce Markdown, but likely would produce even more if we explicitly asked for it. Perhaps I should add "If your answer includes lists, headings, tables, or emphasis, use Markdown syntax."